view/frontend/templates/button_product_page.phtml

```diff
@@ -207,31 +202,35 @@ if (!$block->isSupportableType()) return;
                         options: JSON.stringify(options)
                     }]
                 };
+
+                var hintsPromise = new Promise(function (resolve, reject) {
+                    if (hints) {
+                        resolve(hints);
+                        return;
+                    }
+                    $.get(settings.get_hints_url)
```

Comment:
Do you think we can (in separate PR) find a way how to call the server only if a user logged in?

Reply:
@danac-gs I'd like to create asana about it if you don't mind.